### PR TITLE
[ThreatConnect & Native] Added To Ignore List

### PR DESCRIPTION
**Pull request by shmuel44**
## Related Issues
related: [link](https://jira-dc.paloaltonetworks.com/browse/CIAC-11146) to the issue

## Description
After updating the Docker image in `ThreatConnectV3` to Python 3.11,
the following unit tests are failing: `test_create_context` and `test_create_header`.

### CODE REVIEW

File: Tests/docker_native_image_config.json [hunk 1]
```diff
@@ -146,7 +146,17 @@
             "ignored_native_images": [
                 "native:8.6"
             ]
+        },
+        {
+            "id": "ThreatConnect v3",
+            "reason": "",
```
Comment (shmuel44) [inline Tests/docker_native_image_config.json:152]:
```suggestion
            "reason": "CIAC-11146",
```

**Review by dorschw — APPROVED** [review]

File: Tests/docker_native_image_config.json [hunk 2]
```diff
@@ -146,7 +146,17 @@
             "ignored_native_images": [
                 "native:8.6"
             ]
+        },
+        {
+            "id": "ThreatConnect v3",
+            "reason": "CIAC-11146",
+            "ignored_native_images": [
+                "native:8.6",
+                "native:dev",
```
Comment (JudahSchwartz) [inline Tests/docker_native_image_config.json:155]: isnt native dev also 3.11?

**Review by JudahSchwartz — APPROVED** [review]

File: Tests/docker_native_image_config.json (hunk 2 above)
Comment (shmuel44) [reply Tests/docker_native_image_config.json:155]: correct, this is the reason we add it to ignore list

Comment (JudahSchwartz) [reply Tests/docker_native_image_config.json:155]: mah? its 3.11 in this image and native, so whats the mismatch?

**Comment by shmuel44** [comment]
Closing this PR as PR #35432 will fix the issue.

**Pull request closed without merging**